runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkProcessContext.java

```diff
@@ -125,8 +164,50 @@ public InputT element() {
 
```

Comment:
Side note: line 152 (not part of this diff) has a spelling error of `AggregatprInputT`